File: pbweberog/pbweberog-jar/src/main/java/it/csi/pbandi/pbweberog/integration/request/GetDatiCalcolatiRequest.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbweberog.integration.request;

import java.io.Serializable;

import it.csi.pbandi.pbweberog.dto.erogazione.ErogazioneDTO;

public class GetDatiCalcolatiRequest implements Serializable{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private Long idProgetto;
	private ErogazioneDTO erogazioneDTO;
	public Long getIdProgetto() {
		return idProgetto;
	}
	public void setIdProgetto(Long idProgetto) {
		this.idProgetto = idProgetto;
	}
	public ErogazioneDTO getErogazioneDTO() {
		return erogazioneDTO;
	}
	public void setErogazioneDTO(ErogazioneDTO erogazioneDTO) {
		this.erogazioneDTO = erogazioneDTO;
	}
	
	
}

File: pbweberogwcl/src/app/erogazione/commons/requests/get-dati-calcolati-request.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

/* tslint:disable */
import { ErogazioneDTO } from '../dto/erogazione-dto';
export interface GetDatiCalcolatiRequest {
  erogazioneDTO?: ErogazioneDTO;
  idProgetto?: number;
}
